.storybook/preview.js

```diff
@@ -28,6 +28,8 @@ export const parameters = {
         "Design Tokens",
         "Style",
         "Components",
+        "Utilities",
+        ["Formatters", "Hooks"],
```

Comment:
This configures the sort order for stories and sections in the storybook sidebar